src/components/Modal/Modal.tsx

```diff
@@ -82,6 +89,12 @@ export const useStyles = makeStyles(
         maxHeight: 'unset',
       },
     },
+    contentSize2: {
+      maxHeight: 'calc(100vh - 9rem)',
+      '@media screen and (max-width: 480px)': {
```

Comment:
Where does 480 come from? Should it be based off of one of the breakpoints? If so, should we be using a breakpoint function here so it gets the sizes from the theme?

Reply:
Indeed.  This PR was my attempt to make as few changes as possible to give us what we needed.  480 was previously hard-coded.  We got together yesterday and decided to do a more extensive, non-backward compatible upgrade on this component with new prop names as well as themed breakpoints.